src/layer-manager/lib/generators/carto/carto.js

```diff
@@ -0,0 +1,66 @@
+import layersDirectory from './carto-layers.json'
+
+export const CARTO_TYPE = 'carto'
+
+export const CARTO_FISHING_MAP_API = 'https://carto.globalfishingwatch.org/user/admin/api/v1/map'
```

Comment:
Maybe this should be an external parameter. If we want to do that we need a way to specify configuration from outside, at the generator level, ie:
```
<LayerManager
    generators={[Context(CARTO_FISHING_MAP_API), Basemap]}
    layers={this.state.layers}
>